src/Exceptions/RetroAchievementsException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RetroAchievements\Api\Exceptions;
+
+interface RetroAchievementsException
```

Comment:
All exceptions implement this, which allows for catching any RA client exception. 👍🏻